Formula/zookeeper.rb

```diff
@@ -38,7 +38,9 @@ def shim_script(target)
 
   def default_zk_env
     <<-EOS.undent
-      export ZOOCFGDIR="#{etc}/zookeeper"
+      if [ -z ${ZOOCFGDIR+x} ]; then
```

Comment:
This is one of those weird shell edge case best practice things where it can't tell the difference between unset and empty string unless you go the extra mile.

See [this stackoverflow](http://stackoverflow.com/a/138648290 for in depth details.